lib/importproject.cpp

```diff
@@ -232,111 +232,165 @@ static std::string readUntil(const std::string &command, std::string::size_type
     return ret;
 }
 
+static std::string removeBackslash(std::string str)
+{
+    str.erase(std::remove(str.begin(), str.end(), '\\'), str.end());    
+    return str;
+}
+
+static void skip_whitespaces(const std::string& command, std::string::size_type *pos)
+{
+    while (*pos < command.size() && command[*pos] == ' ')
+        (*pos)++;
+}
+
+
+static std::string parseTillNextCommandOpt(const std::string& command, std::string::size_type *pos)
+{
+    std::string ret{};
+    while (*pos < command.size() && command[*pos] != '/' && command[*pos] != '-' )
+    {
+        (*pos)++;
+    }
+    (*pos)++;
+    if (*pos >= command.size())
+    {
```

Comment:
stylistic nit; we do not usually use redundant `{}` around a single statement. And we usually do not write `{` on its own line. Well this would be corrected by our `runastyle` tool later but it looks nice if it follows normal cppcheck syntax directly.

Reply:
> And we usually do not write { on its own line. Well this would be corrected by our runastyle tool later but it looks nice if it follows normal cppcheck syntax directly.

You can ignore that for now. It will be fixed by runastyle later.

But well feel free to try out the runastyle script now immediately. You need to install astyle 3.0.1 , the exact version is very important to avoid formatting conflicts.

Reply:
Thx, yes I'm aware that the style is not 100% correct, on some places I also only used 2 instead of 4 spaces and so on.
I'm just used to different styling...
But I did not want to waste to much time in cleanup when the general solution would not fit.

So I will clean it up, promise🤞